app/pods/publications/index/template.hbs

```diff
@@ -302,7 +302,7 @@
 
         {{#if this.filterTableColumnOptionKeys.speedProcedureFilterOption}}
           <td class="auk-u-text-align--center">
-            {{#if row.hasPriority}}
+            {{#if row.hasUrgencyLevel}}
```

Comment:
De `hasUrgencyLevel` is hier wat overbodig. Je kan ook gewoon checken op `{{row.urgencyLevel}}` in de template. De `hasUrgencyLevel` getter in het model kan dan ook verwijderd worden